packages/compass-explain-plan/src/components/explain-tree/explain-tree.tsx

```diff
@@ -59,10 +59,11 @@ const ExplainTree: React.FunctionComponent<ExplainTreeProps> = ({
       data={root}
       getNodeSize={getNodeSize}
       getNodeKey={getNodeKey}
-      linkColor={darkMode ? palette.gray.base : palette.gray.light2}
+      linkColor={darkMode ? palette.gray.dark2 : palette.gray.light2}
+      arrowColor={darkMode ? palette.gray.base : palette.gray.light1}
       linkWidth={6}
       horizontalSpacing={spacing[6]}
-      verticalSpacing={spacing[6]}
+      verticalSpacing={spacing[5] + 8}
```

Comment:
This is bigger than the designs, if we are using custom numbers we might as well just set the number exactly to what is provided there:

![image](https://github.com/mongodb-js/compass/assets/5036933/8dd2d97f-dc6c-405b-a5ea-a1add1bec27a)

Reply:
> if we are using custom numbers we might as well just set the number exactly to what is provided there (or to spacing[5] as it's very close to the expected spacing)

Let's try to do that also if it was in the design :), we can ask if it was intentional to have a custom space that is off sync with the tokens